exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -249,16 +282,64 @@ func (g *gaugeConsumer) PushInternalMetrics(errs *[]error) {
 	}
 }
 
-func (g *gaugeConsumer) pushSingleNumberDataPoint(
-	metric pdata.Metric, numberDataPoint pdata.NumberDataPoint, errs *[]error) {
+type sumConsumer struct {
+	sender                senders.MetricSender
+	logger                *zap.Logger
+	missingValues         counter
+	reportInternalMetrics bool
+}
+
+func newSumConsumer(sender senders.MetricSender, options *consumerOptions) typedMetricConsumer {
+	var fixedOptions consumerOptions
+	if options != nil {
+		fixedOptions = *options
+	}
+	fixedOptions.replaceZeroFieldsWithDefaults()
+	return &sumConsumer{
+		sender:                sender,
+		logger:                fixedOptions.Logger,
+		reportInternalMetrics: fixedOptions.ReportInternalMetrics,
+	}
+}
+
+func (s *sumConsumer) Type() pdata.MetricDataType {
+	return pdata.MetricDataTypeSum
+}
+
+func (s *sumConsumer) Consume(metric pdata.Metric, errs *[]error) {
```

Comment:
The specification [gives some indications on how Sums are mapped to a timeseries model](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/metrics/datamodel.md#opentelemetry-protocol-data-model) (PRW-based), in which monotonic Sums are mapped as counters while non-monotonic sums are mapped as gauges. 

What to do here ultimately depends on your timeseries model (which I don't know), so this implementation is not necessarily wrong, but I wanted to bring it up here just in case it helped

Reply:
Thank you for pointing this out. Tanzu observability has a special API call to send a delta sum metric. To send a cumulative sum metric to wavefront, you use the same API call that you would use to send a gauge metric.

Reply:
Thank you for pointing this out. Tanzu observability has a special API call to send a delta sum metric. To send a cumulative sum metric to wavefront, you use the same API call that you would use to send a gauge metric.